src/commands/functions/functions-create.js

```diff
@@ -189,6 +203,51 @@ const pickTemplate = async function ({ language: languageFromFlag }) {
 
 const DEFAULT_PRIORITY = 999
 
+const selectTypeOfFunc = async () => {
+  const functionTypes = [
+    { name: 'Edge Function (Deno)', value: 'edge' },
+    { name: 'Serverless Function (Node/Go)', value: 'serverless' },
+  ]
+
+  const { functionType } = await inquirer.prompt([
+    {
+      name: 'functionType',
+      message: "Select the type of function you'd like to create",
+      type: 'list',
+      choices: functionTypes,
+    },
+  ])
+  return functionType
+}
+
+const ensureEdgeFuncDirExists = function (command) {
+  const { config, site } = command.netlify
+  const siteId = site.id
+  let functionsDirHolder = config.build.edge_functions
+
+  if (!siteId) {
+    error(`${NETLIFYDEVERR} No site id found, please run inside a site directory or \`netlify link\``)
+  }
+
+  if (!functionsDirHolder) {
+    log(`${NETLIFYDEVLOG} Edge Functions directory not specified in netlify.toml`)
```

Comment:
`config.build.edge_functions` being defined and the directory being set in the TOML aren't related. That config property will be populated with the default directory if you're using edge functions in your project. Just calling this out because it might confuse folks who read this code in the future.

Personally, I don't think we need this log line, because I'm not sure it provides useful or actionable information to the user.

Reply:
Oh yeah, that makes sense. Removed!